changehc/delphi_changehc/run.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""Functions to call when running the function.
+
+This module should contain a function called `run_module`, that is executed
+when the module is run with `python -m delphi_changehc`.
+"""
+
+# standard packages
+import logging
+from datetime import datetime, timedelta
+from pathlib import Path
+
+#  third party
+from delphi_utils import read_params
+
+# first party
+from .update_sensor import CHCSensorUpdator
+
+
+def run_module():
+    """Run the delphi_changehc module.
+    """
+
+    params = read_params()
+
+    logging.basicConfig(level=logging.DEBUG)
+
+    ## get end date from input file
+    # the filenames are expected to be in the format:
+    # Denominator: "YYYYMMDD_All_Outpatients_By_County.dat.gz"
+    # Numerator: "YYYYMMDD_Covid_Outpatients_By_County.dat.gz"
+    
+    if params["drop_date"] is None:
```

Comment:
is it worth testing some of this functionality? just to make sure that all the right files/dates get generated

Reply:
The params are loaded straight from `params.json` so I don't think this can be tested very easily. There is similar code for the hosp signal and this functionality wasn't tested.